src/cascade/executor/estimate_location.py

```diff
@@ -277,18 +284,20 @@ def _fit_and_predict_fixed_effect_sample(sim_model, sim_data, fit_file, location
                 _fit_and_predict_fixed_effect_sample,
                 sim_model,
                 sim_data,
-                fit_file,
+                base_path / fit_file,
                 input_data.locations_df,
                 model.location_id,
                 local_settings,
                 draw_idx
             ))
 
         results = await asyncio.gather(*jobs)
-    return results
+    return [r for r in results if r is not None]
```

Comment:
I wouldn't have guessed that.

Reply:
Hmm? I have to return something from the jobs, even when the fail, so here I filter the failures out.